builds/e2e/e2e.yaml

```diff
@@ -8,14 +8,14 @@ resources:
   pipelines:
   - pipeline: images
     source: 'Azure-IoT-Edge-Core Build Images'
-    branch: master
+    branch: 'release/1.2'
```

Comment:
can I ask why this change? that is where the pipeline definition is coming from, not the trigger bransh, AFAIK

Reply:
Yeah. I think we want to this pipeline to be using yaml from this branch right?

Reply:
This isn't the trigger branch, this is a filter on grabbing artifacts from "Azure-IoT-Edge-Core Build Images" builds. So this will cause the end-to-end test pipeline to use artifacts from the latest successful "Build Images" build _against the release/1.2 branch_.

Reply:
Thank you for the clarification! I think this is what we want here.